addons/website_sale_slides/tests/test_course_purchase_flow.py

```diff
@@ -5,6 +5,15 @@
 
 
 class TestCoursePurchaseFlow(common.SlidesCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.salesman = cls.env['res.users'].create({
+            'name': 'salesman',
+            'login': 'salesman',
+            'email': 'salesman007@example.com',
+            'groups_id': [(6, 0, cls.env.ref('sales_team.group_sale_salesman').ids)],
+        })
```

Comment:
Hello @abz-odoo 

Rename to "cls.user_salesman" to match existing naming convention (e.g: existing "cls.user_officer" variable).

You can add a blank like after the setUpClass method.

Also, did you check if this fix had to be done in an earlier stable than 14.0 ?

Reply:
my bad, thanks for advices

new pr in 13.0: https://github.com/odoo/odoo/pull/75824